CONTRIBUTING.md

```diff
@@ -1,19 +1,18 @@
-## Git Considerations
+# Contributing
+Contributors are encouraged to be a [member](https://www.openhwgroup.org/membership/) of the
+OpenHW Group.  New members are always welcome.
 
-- Create your branch to commit your changes and then create a Pull Request.
-- Separate subject from body with a blank line.
-- Limit the subject line to 50 characters.
-- Capitalize the subject line.
-- Do not end the subject line with a period.
-- Use the imperative mood in the subject line.
-- Use the present tense ("Add feature" not "Added feature").
-- Wrap the body at 72 characters.
-- Use the body to explain what and why vs. how.
+## Getting Started
+The [OpenHW Work Flow](https://github.com/openhwgroup/core-v-docs/blob/master/verif/Common/OpenHWGroup_WorkFlow.pdf) document
```

Comment:
same is true for other repositories. because i copied the file, e.g. https://github.com/openhwgroup/cv32e40p/blob/master/CONTRIBUTING.md

for a widely copied file with a bunch of references, it would be good if the referenced file locations don't change.
One solution is to also refer to a concrete revision of the repository containing the file, but then updates do not automatically propagate. An alternative could be to use symlinks to make sure links keep working after moving the files. 
I will not take action now, until there is a decision of how to do this across repositories. 
If this is the only issue, my advice would be to merge with the broken link and to create a new ticket, as we can solve this later and it is sort of unrelated.

Reply:
done